src/Sylius/Behat/Page/Cart/CartSummaryPage.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Cart;
+
+use Sylius\Behat\Page\SymfonyPage;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class CartSummaryPage extends SymfonyPage
+{
+    public function openPage()
```

Comment:
I don't think you need custom method for visiting page, you have already provided route name, so simple call ``->open()`` on your page object.